src/platform/site-wide/side-nav/components/NavItem.js

```diff
@@ -0,0 +1,98 @@
+// Dependencies
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { get } from 'lodash';
+// Relative
+import DuplicateLineLabel from './DuplicateLineLabel';
+import ExpandCollapseButton from './ExpandCollapseButton';
+import LabelText from './LabelText';
+
+const NavItem = ({
+  depth,
+  item,
+  index,
+  renderChildItems,
+  sortedNavItems,
+  toggleItemExpanded,
+}) => {
+  // Derive the item properties.
+  const expanded = get(item, 'expanded');
+  const id = get(item, 'id');
+  const isSelected = get(item, 'isSelected');
+  const label = get(item, 'label', '');
+
+  // Derive the depth booleans.
+  const isFirstLevel = depth === 1;
+  const isDeeperThanSecondLevel = depth >= 2;
+
+  // Caclculate the indentation for the child items.
+  const indentation = isDeeperThanSecondLevel ? 20 * (depth - 1) : 20;
+
+  // Determine if we should show a line at the end.
+  const showLine = isFirstLevel && index !== sortedNavItems.length - 1;
+
+  return (
+    <li className={`va-sidenav-level-${depth}`} key={id}>
+      <div
+        aria-label={label}
+        className={classNames({
+          'va-sidenav-item-label': true,
+          selected: isSelected,
+        })}
+        onClick={toggleItemExpanded(id)}
+        style={{ paddingLeft: indentation }}
+      >
+        {/* Label */}
+        <LabelText item={item} />
+
+        {/* Expand/Collapse Button */}
+        <ExpandCollapseButton depth={depth} item={item} />
+      </div>
+
+      {/* Duplicate Line + Label when Expanded */}
+      <DuplicateLineLabel depth={depth} item={item} />
```

Comment:
Lol at this component name, `DuplicateLineLabel`!

Reply:
Well, I mean, it is just a straight up duplicate nav item label! haha